tensorflow/lite/kernels/test_util.h

```diff
@@ -233,15 +233,25 @@ class SingleOpModel {
   // Quantize and populate data for bias with per channel quantization.
   void PerChannelQuantizeBias(int index, const std::vector<float>& input_data) {
     const int32_t num_inputs = input_data.size();
-    std::vector<int32_t> quantized_output(num_inputs);
     TfLiteTensor* t = interpreter_->tensor(index);
     auto* params =
         reinterpret_cast<TfLiteAffineQuantization*>(t->quantization.params);
-    for (int i = 0; i < num_inputs; ++i) {
-      quantized_output[i] = input_data[i] / params->scale->data[i];
+    CHECK(t->type==kTfLiteInt32 || t->type==kTfLiteInt64);
+    if (t->type==kTfLiteInt32) {
```

Comment:
Can you run the clang formatter on this CL? There are a number of spacing/indent cases that that should fix, see https://www.tensorflow.org/community/contribute/code_style#c_coding_style. Thanks!